frontend/src/game/Virus.ts

```diff
@@ -0,0 +1,32 @@
+import { GameObjects, Physics, Scene } from "phaser";
+import { EventBus } from "./EventBus";
+import ClickableBit from "./ClickableBit";
+import { ChildProcess } from "child_process";
+
+export default class computerVirus extends GameObjects.Sprite {
+  testVar: number;
```

Comment:
You can remove this.

Reply:
I removed all the unneeded code here.

Reply:
I'm working on a commit to delete `testVar` and refactor other stuff.